_content/applications/manager/index.md

```diff
@@ -33,4 +33,4 @@ To demonstrate the usage of this API we have an [example in Go](https://github.c
 
 ## SDKs
 
-Only the [Java SDK](https://github.com/TheThingsNetwork/java-app-sdk) has experimental support for the Application Manager API.
+The [Java SDK](https://github.com/TheThingsNetwork/java-app-sdk/tree/2.1.0) (version 2.1.0) has support for the Application Manager API.
```

Comment:
I asked @cambierr what he prefered. We agree with you, I changed it to the `master` branch :smile: